src/QMCWaveFunctions/EinsplineSetBuilderCommon.cpp

```diff
@@ -138,7 +138,7 @@ EinsplineSetBuilder::BroadcastOrbitalInfo()
   int numIons = IonTypes.size();
   int numAtomicOrbitals = AtomicOrbitals.size();
   int numDensityGvecs = TargetPtcl.DensityReducedGvecs.size();
-  PooledData<RealType> abuffer;
+  PooledData<double> abuffer;
```

Comment:
I think this buffer type should be parameterized. Same for the other declarations. Why was it changed?

Reply:
This is in builder to send receive meta data info. Some infos are floats and some are doubles. In order not to loose precision, using double type PooledData is safe. This is local and I don't want to entangle it with others. Instead in wavefunction, typedef WFBufferType is used to have a consistent control.

Reply:
I created an issue for the documentation task. I need to learn and practice a bit doxygen style before doing it.